com.unity.robotics.urdf-importer/Tests/Runtime/UrdfComponents/UrdfJoints/UrdfJointContinuousTests.cs

```diff
@@ -0,0 +1,137 @@
+using NUnit.Framework;
+using UnityEngine;
+using RosSharp.Urdf;
+using Joint = RosSharp.Urdf.Joint;
+
+public class TestUrdfJointContinuous : UrdfJointContinuous
+{
+    public void TestImportJointData(Joint joint)
+    {
+        unityJoint = gameObject.GetComponent<ArticulationBody>();
+        ImportJointData(joint);
+    }
+
+    public Joint TestExportSpecificJointData(Joint joint)
+    {
+        unityJoint = gameObject.GetComponent<ArticulationBody>();
+        return ExportSpecificJointData(joint);
+    }
+
+    public void SetAxisOfMotion(Vector3 axisofMotion) => this.axisofMotion = axisofMotion;
+    public void Dynamics(Joint.Dynamics dynamics) => SetDynamics(dynamics);
+}
+
+public class UrdfJointContinuousTests
+{
+    [Test]
+    public void Create_Succeeds()
```

Comment:
general question: does it better follow convention if this were `Create_ContinuousJoint_Succeeds()` or would calling out specifically `Continuous` be unnecessary given that these are all Continuous tests?

Reply:
Good point. I am not exactly sure but feel that `Create_ContinuousJoint_Succeeds` might be better. Will update it.